<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageIntrfSettingsWgt</class>
 <widget class="QWidget" name="PageIntrfSettingsWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PageIntrfSettingsWgt</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="AttenuatorSettBox">
     <property name="title">
      <string>Attenuator Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="WaveAtBox">
        <property name="title">
         <string>Attenuator type</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rBoneWheel">
           <property name="text">
            <string>One Wheel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>186</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLineEdit" name="lineWav">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="scale_lab">
           <property name="text">
            <string>nm</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rBtwoWheel">
           <property name="text">
            <string>Two Wheels</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>111</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="wave1but">
           <property name="text">
            <string>1055 nm</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="wave2but">
           <property name="text">
            <string>532 nm</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QRadioButton" name="wave3but">
           <property name="text">
            <string>262 nm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="Wheel1Box">
        <property name="title">
         <string>Wheel1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="Wheel2Box">
        <property name="title">
         <string>Wheel2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
